flask_monitoringdashboard/core/config/parser.py

```diff
@@ -23,9 +23,14 @@ def parse_version(parser, header, version):
             # The file is specified by: 'ref: <location>'
             git_file = (open(os.path.join(git, 'HEAD')).read().rsplit(': ', 1)[1]).rstrip()
             # read the git-version
-            version = open(git + '/' + git_file).read()
-            # cut version to at most 6 chars
-            return version[:6]
+            version_file = os.path.join(git , git_file)
```

Comment:
it's not clear why this is part of the diff... it is not code written by you @ayanlee ... is it?

Reply:
ah. now i see. it's a PR to master. 
try to PR to development instead!

Reply:
My bad! I'll fix it quick.